src/main.ts

```diff
@@ -103,12 +125,42 @@ function watch(config: webpack.Config, options: WebpackOptions, args: BuildArgs)
 	})(config.entry);
 
 	const compiler = webpack(config);
+	const portRange = String(args.port || '9999:9990');
+	let ports: number[] = [];
+	let serverPort: number | undefined;
+
+	if (portRange.indexOf(':') >= 0) {
+		let [low, high] = portRange.split(':').map(p => parseInt(p, 10));
```

Comment:
should wrap argument in parenthesises, such as `((p) => parseInt(p, 10))`, also can do better at naming for `p`?